src/helpers/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const customPageView = (url: string) => {
 
 export const customOutboundLink = (url: string) =>
   develop
-    ? window.open(url, "_blank")
+    ? window.open(url, "_blank", 'noopener')
```

Comment:
On chrome when clicking on an issue it opens a new pop-up window instead of a tab, is there a way to avoid the pop-up? (It works perfectly of firefox). Something like this https://stackoverflow.com/questions/46147949/using-rel-noopener-in-window-open might be a solution